ui/src/app/Login/FixedUsernameLoginForm.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
```

Comment:
Is this re-implementing the whole login form?

Reply:
Yes it is.  I have very rudimentary react knowledge, so perhaps there is a better way.  I couldn't find any way to modify the encapsulated component (loginform) from outside, so did it by copying/modifying it.  Seems like you can inherit/extend, but again I didn't see how to modify already included components by doing that.  Open to any suggestion.

Reply:
thats the only way to do it. I tired many ways but you would have to recreate the form.

Reply:
I figured that you tried to do that as well, thanks.